gtsfm/utils/metrics.py

```diff
@@ -141,15 +128,15 @@ def compute_translation_angle_metrics(
     for (i1, i2) in i2Ui1_dict:
         i2Ui1 = i2Ui1_dict[(i1, i2)]
         angles.append(comp_utils.compute_translation_to_direction_angle(i2Ui1, wTi_list[i2], wTi_list[i1]))
-    return compute_errors_statistics(angles)
+    return GtsfmMetric("translation_angle_deg", angles)
 
 
 def compute_averaging_metrics(
     i2Ui1_dict: Dict[Tuple[int, int], Unit3],
     wRi_list: List[Optional[Rot3]],
     wti_list: List[Optional[Point3]],
     gt_wTi_list: List[Optional[Pose3]],
-) -> Dict[str, StatsDict]:
+) -> GtsfmMetricsGroup:
```

Comment:
i prefer to call this `compute_pose_error_metrics()`, instead of `compute_averaging_metrics()`, since this could be called on both BA output or averaging output, alike.

Reply:
TODO in future PR